ouf.lua

```diff
@@ -262,13 +243,31 @@ local function updateRaid(self, event)
 	end
 end
 
+-- boss6-8 exsist in some encounters, but unit event registration seems to be
+-- completely broken for them, so instead we use OnUpdate to update them.
+local eventlessUnits = {
+	['boss6'] = true,
```

Comment:
`boss6 = true` would be more consistent with the current code style. See https://github.com/oUF-wow/oUF/blob/master/events.lua#L19-L29. On the other hand we break this in tags.lua, so feel free to ignore this.